spec.md

```diff
@@ -461,28 +467,35 @@ As an extension to RFC 9078, it is allowed to send empty reaction message,
 in which case all previously sent reactions are retracted.
 
 
-# Miscellaneous
+# Attaching a contact to a message
 
-Messengers SHOULD use the header `In-Reply-To` as usual.
+Messengers MAY allow the user to attach a contact to a message
+in order to share it with the chat partner.
+The contact MUST be sent as a [vCard](https://datatracker.ietf.org/doc/html/rfc6350).
 
-Messengers MAY send and receive Message Disposition Notifications
-(MDNs, [RFC 8098](https://tools.ietf.org/html/rfc8098),
-[RFC 3503](https://tools.ietf.org/html/rfc3503))
-using the `Chat-Disposition-Notification-To` header
-instead of the `Disposition-Notification-To`
-(which unfortunately forces many other MUAs
-to send weird mails not following any standard).
+The vCard MUST contain `EMAIL`
+and `VERSION` (which version of the vCard standard you're using),
+and if available it SHOULD contain `FN` (display name),
+`PHOTO` (avatar),
+`KEY` (public key), and
```

Comment:
could you say in which format the key must be?

Reply:
It's in base64, in our case we obtain it by calling `to_base64()` on the `pgp` crate's `SignedPublicKey`. Is there a way to further specify the key format than saying it's base64?

Reply:
I can't find a `to_base64` in [SignedPublicKey](https://docs.rs/pgp/latest/pgp/composed/signed_key/struct.SignedPublicKey.html). Can you help me understand what function is used in the conversion?

And/or: Does the format look different from normal "ASCII Armored" OpenPGP keys?

Reply:
> I can't find a `to_base64` in [SignedPublicKey](https://docs.rs/pgp/latest/pgp/composed/signed_key/struct.SignedPublicKey.html). Can you help me understand what function is used in the conversion?

It's implemented by the `deltachat::key::DcKey` trait.

Reply:
Thanks! So it's [this code](https://github.com/deltachat/deltachat-core-rust/blob/ac198b17bff217b898f89841a51f0136bf9b0a88/src/key.rs#L52-L66).

The code takes the [SignedPublicKey](https://docs.rs/pgp/latest/pgp/composed/signed_key/struct.SignedPublicKey.html), and serialized it as a "binary" OpenPGP object (not ASCII Armored), and then base64 encodes that.

I don't know what the best language is, for the `spec.md` documentation. But I'd suggest something along the lines: "An OpenPGP public key, in binary format, encoded with vanilla base64 (note that this is different from the OpenPGP 'ASCII Armor' format)".

Reply:
I updated the documentation, thanks!

I also checked that the key in the vCard created by protonmail can be imported successfully. So, apparently we're doing everything correctly :+1:

Reply:
Nice! :)